src/ui/public/state_management/state_storage/hashing_store.js

```diff
@@ -49,25 +42,12 @@ class HashingStore {
    *  @return {string} the hash of the value
    */
   hashAndSetItem(object) {
-    const json = angular.toJson(object);
+    const json = JSON.stringify(object);
```

Comment:
From the [docs](https://docs.angularjs.org/api/ng/function/angular.toJson) it looked like `angular.toJson` was meant mostly from stripping out angular $$-prefixed properties. I think we can remove that dependency and use the native JSON here.